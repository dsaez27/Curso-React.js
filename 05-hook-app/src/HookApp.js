import React from 'react'
//? rafc para crear componente rapido

export const HookApp = () => {
    return (
        <>
            <h1>Hola mundo</h1>
        </>
    )
}

