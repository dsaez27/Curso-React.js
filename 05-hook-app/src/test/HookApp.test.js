import React from 'react';
import { shallow } from 'enzyme';
import { HookApp } from '../HookApp';

describe('Pruebas en <HookApp />', () => {
    test('Debe de mostrarse <HookApp /> correctamente', () => {
        const wrapper = shallow(<HookApp />);
        expect(wrapper).toMatchSnapshot();
    });
});
