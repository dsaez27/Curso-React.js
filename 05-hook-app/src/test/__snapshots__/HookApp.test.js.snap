// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HookApp /> Debe de mostrarse <HookApp /> correctamente 1`] = `
<Fragment>
  <h1>
    Hola mundo
  </h1>
</Fragment>
`;
